blocks/library/gallery/block.scss

```diff
@@ -0,0 +1,39 @@
+
+.blocks-gallery {
```

Comment:
Yeah, that sounds like a good idea, same apprehension from the above comment applies here.